data/core/emptyview.lua

```diff
@@ -6,6 +6,10 @@ local View = require "core.view"
 ---@field super core.view
 local EmptyView = View:extend()
 
+function EmptyView:get_name()
+  return "Get Started"
+end
+
```

Comment:
> As a side effect, it also says `Get Started` when nothing is opened now.

Horrible workaround, but still...
```suggestion
function EmptyView:get_name()
  return "Get Started"
end

function EmptyView:get_filename()
  return ""
end
```